src/config.rs

```diff
@@ -98,6 +98,9 @@ pub struct Config {
 
     /// Configuration for the sockets to listen on.
     pub listen_config: ListenConfig,
+
+    /// Flag that enables notifying the application layer of unverifiable ENRs, peers that are denied sessions. Defaults to `false`.
+    pub notify_unverifiable_enr: bool,
```

Comment:
Appreciate the concern to make it optional to add EL specific features, 

But I think in this case, its just an event. The event-stream is optional for users, so they can just not listen to the event stream if they don't want. 

I dont think we need a config param, lets just make it always on, which saves a bit of code and complexity.